README.md

```diff
@@ -264,8 +264,11 @@ Clone the openrouteservice-js repository from GitHub into a development environm
 git clone https://github.com/GIScience/openrouteservice-js.git
 cd openrouteservice-js
 
-#Install the dependencies
+# Install the dependencies
 npm install
+
+# Make your openrouteservice API key available
+sh setup.sh your-api-key
```

Comment:
would using < your api key > instead of your-api-key be more in line with conventional documentation practices?

Reply:
Yes, that would make it clearer. Done.